# Generated by Django 5.1.5 on 2025-03-19 07:22

import django.utils.timezone
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('logs', '0012_alter_logserror_info'),
    ]

    operations = [
        migrations.CreateModel(
            name='LogsOrderError',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('date', models.DateField(default=django.utils.timezone.now, verbose_name='Дата добавления')),
                ('type_error', models.CharField(choices=[('file_structure_error', 'фаил не соответствует нужной структуре'), ('file_api_error', 'ошибка при загрузке с api'), ('error', 'Ошибка')], default='error', max_length=20)),
                ('location', models.CharField(blank=True, max_length=200, null=True, verbose_name='Место ошибки')),
                ('info', models.CharField(blank=True, max_length=5000, null=True, verbose_name='Инфо о ошибке')),
            ],
            options={
                'verbose_name': 'Логи фоновые по сделкам',
                'verbose_name_plural': 'Логи фоновые по сделкам',
            },
        ),
    ]
